src/pages/folder.js

```diff
@@ -0,0 +1,212 @@
+import AddLink from '../components/addLink';
+import Footer from '../components/footer';
+import GNB from '../components/gnb';
+import styled from 'styled-components';
+import SearchBar from '../components/searchBar';
+import { useEffect, useState } from 'react';
+import { getLinkData } from '../api/api';
+import Menu from '../components/menu';
+import plusIcon from '../assets/plusIcon.svg';
+import Card from '../components/card';
+import shareIcon from '../assets/share.svg';
+import penIcon from '../assets/pen.svg';
+import deleteIcon from '../assets/delete.svg';
+
+function Folder() {
+  const [data, setData] = useState();
+  const [selectedItem, setSelectedItem] = useState('전체');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await getLinkData();
+        setData(response);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  return (
+    <FolderMain>
+      <GNB isStickyDisabled={false} />
+      <FolderFrame>
+        <AddLink />
+      </FolderFrame>
+      <FolderContent>
+        <SearchBar />
+        <MenuArea>
+          <Menu selectedItem={selectedItem} setSelectedItem={setSelectedItem} />
+          <AddFolder>
+            폴더 추가 <PlusImg src={plusIcon} />
+          </AddFolder>
+        </MenuArea>
+        <FolderTitleArea>
+          <FolderTitle>{selectedItem}</FolderTitle>
+          {selectedItem === '전체' ? (
+            <></>
```

Comment:
아무것도 렌더링하고 싶지 않을 떄는 `return null` 하시면 됩니다!

```suggestion
            null
```